client/app/components/sections/Topbar/Topbar.css

```diff
@@ -52,10 +54,19 @@
     font-size: var(--Menu_fontSizeSmall);
   }
 
+  & > * {
+    margin-right: 24px;
```

Comment:
This `24px` is repeated in several places and could be a variable to convey its meaning.